packages/server/src/serviceworker.ts

```diff
@@ -29,6 +30,12 @@ export class ServiceWorkerRegistrationWrapper
   }
 
   private async initialize() {
+    if (this._config['disabled']) {
```

Comment:
Normally the following notation is preferred when the object is properly typed:

```suggestion
    if (this._config.disabled) {
```